rewrite(docs): structure/layout + improved docs (#80)
spicetify/spicetify-docs: docs/addons/themes.md

description: ✨ Themes for Spicetify.
---

> The theming heroes of the Spicetify community have created a huge library of themes which can be found in the following repositories:

Reviewer: Add some wording to say that this is not a complete exhaustive list and there may be other themes not listed.

Reviewer 2: ```suggestion
The Spicetify community has created a huge library of themes which can be found in the following repositories:
```
Let's remove the "theming heroes" 😅
